admin/team-principles.md

```diff
@@ -0,0 +1,8 @@
+# Works on My Machine -- Team Code of Conduct
+1. Be respectful towards your teammates at all times.
+2. Attend meetings on a best-effort basis.
+3. Give earliest possible notice when unable to attend a meeting, and read meeting minutes afterwards.
+4. Check Slack at least once a day and be responsive to pings.
+5. Respectful debate of design decisions is encouraged, but as a last resort will be arbitrated by a plurality vote. 
+6. Strive for quality over quantity of features at all times.
+7. Proactively update GitHub issues whenever working on anythign project-related.
```

Comment:
The 5th point could be re-written as "Respectful debate of design decisions is encouraged, however, as a last resort, it will be arbitrated by a plurality vote."
Typo on the 7th point, "Anything" is misspelled as "anythign".